source/op/pt/CMakeLists.txt

```diff
@@ -20,6 +20,16 @@ endif()
 
 find_package(MPI)
 if(MPI_FOUND)
+  include(CheckCXXSymbolExists)
+  set(CMAKE_REQUIRED_INCLUDES ${MPI_INCLUDE_PATH})
+  set(CMAKE_REQUIRED_LIBRARIES ${MPI_LIBRARIES})
```

Comment:
Both `${MPI_INCLUDE_PATH}` and `${MPI_LIBRARIES}` are deprecated: https://cmake.org/cmake/help/latest/module/FindMPI.html#backward-compatibility